### PR TITLE
Update some instrumentation short-list and new badges

### PR DESCRIPTION
**Pull request by wu-sheng**
1. Add hprose-java and motan to instrumentation short-list.
1. Change badge with version-tag. Get ready for the next spec version.
1. Remove blurry and un-versioned badges.

@bensigelman , thoughts?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,9 @@ The following are all on the OpenTracing instrumentation short-list:
 - [ ] Elasticsearch
 - [x] [Finagle](https://github.com/twitter/finagle/pull/520#issuecomment-249959538)... almost :confused: 
 - [ ] MySQL / mariadb
+- [x] [motan](https://github.com/weibocom/motan/tree/master/motan-extension/filter-extension/filter-opentracing)
+- [ ] hprose:
+  - [x] [Java](https://github.com/opentracing-contrib/java-hprose)...almost, wait for release.
```
Comment (bensigelman) [inline README.md]: I think we can leave the `"...almost, wait for release."` off of this... it'll happen soon, and the pointer is the important part.

Comment (wu-sheng) [inline README.md]: Remove it.

**Comment by wu-sheng** [comment]
I have removed the `badge` folder, and move the badge description to README.md. Also as you said, provide two-type badges.

And this version is definitely OT specification version.

**Comment by pavolloffay** [comment]
Maybe an off topic question:

Wouldn't it be better to sort instrumentations by languages? But then there could be a category/s for databases, or some technology where I don't really care about the language.

**Comment by bensigelman** [comment]
@pavolloffay 

> Wouldn't it be better to sort instrumentations by languages?

Probably :)

File: README.md [hunk 2]
```diff
@@ -34,6 +34,26 @@ The following are all on the OpenTracing instrumentation short-list:
 - [ ] Elasticsearch
 - [x] [Finagle](https://github.com/twitter/finagle/pull/520#issuecomment-249959538)... almost :confused: 
 - [ ] MySQL / mariadb
+- [x] [motan](https://github.com/weibocom/motan/tree/master/motan-extension/filter-extension/filter-opentracing)
+- [ ] hprose:
+  - [x] [Java](https://github.com/opentracing-contrib/java-hprose)
+
+# Badges
+
+Once you instrument your code or project to be enabled for OpenTracing, you can add OpenTracing badge or versioned badge to your project repo.
```
Comment (bensigelman) [inline README.md]: I have some copy edits to suggest, but I can just make them myself post-merge. Sound good?

Comment (wu-sheng) [inline README.md]: Yours are better.😃

**Comment by wu-sheng** [comment]
@pavolloffay  I am not sure `by language` is a good choice. `By tech` may be better. Many rpc frameworks, database drivers, cache clients, which need instrumentations, have multi-languages implemetations.

**Pull request merged**